### PR TITLE
[ADD] purchase_order_line_stock_available module

### PR DESCRIPTION
**Pull request by cubells**
Simple migration of module from https://github.com/odoomrp/odoomrp-utils repository.

This module allows to view the virtual stock quantity of a product in the
purchase order line.

cc @Tecnativa

### CODE REVIEW

File: purchase_order_line_stock_available/models/purchase.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order.line'
+
+    virtual_available = fields.Float(related='product_id.virtual_available')
```
Comment (rousseldenis) [inline purchase_order_line_stock_available/models/purchase.py]: Should be readonly

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Minor comment.

@cubells Do you have tested it in terms of performance with a purchase with lot of lines?

IMHO this is not a migration as the module was not part of OCA before

**Comment by cubells** [comment]
@rousseldenis I've tested the module in a purchase with 237 lines and I have not noticed performance problems.

**Comment by rousseldenis** [comment]
> @rousseldenis I've tested the module in a purchase with 237 lines and And I have not noticed performance problems.

Ok, with a lot of quants and existing moves too ?

**Comment by cubells** [comment]
@rousseldenis yes

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**